packages/adapter-netlify/README.md

```diff
@@ -36,6 +36,24 @@ Then, make sure you have a [netlify.toml](https://docs.netlify.com/configure-bui
 
 It's recommended that you add the `build` and `functions` folders (or whichever other folders you specify) to your `.gitignore`.
 
-## Extra Resources
+## Customizing Setup
 
-Guide for SvelteKit + Netlify Forms users: https://dev.to/swyx/how-to-use-sveltekit-with-netlify-forms-5gmj
+### Using Netlify Redirect Rules
+
+During compilation a required "catch all" redirect rule is automatically appended to your `_redirects` file. (If it doesn't exist yet, it will be created.) That means:
+
+- `[[redirects]]` in `netlify.toml` will never match as `_redirects` has a [higher priority](https://docs.netlify.com/routing/redirects/#rule-processing-order). So always put your rules in the [`_redirects` file](https://docs.netlify.com/routing/redirects/#syntax-for-the-redirects-file).
+- `_redirects` shouldn't have any custom "catch all" rules such as `/* /foobar/:splat`. Otherwise the automatically appended rule will never be applied as Netlify is only processing [the first matching rule](https://docs.netlify.com/routing/redirects/#rule-processing-order).
+
+### Defining Netlify Forms
+
+1. Create your Netlify HTML form as described [here](https://docs.netlify.com/forms/setup/#html-forms), e.g. as `/routes/contact.svelte`. (Don't forget to add the hidden "form-name" input element!)
+2. Tell SvelteKit to [prerender](https://kit.svelte.dev/docs#ssr-and-javascript-prerender) your `contact.svelte` component by adding:
+   ```svelte
+   <script context="module">
+       export const prerender = true;
+   </script>
+   ```
+3. If your Netlify form has a [custom success message](https://docs.netlify.com/forms/setup/#success-messages), e.g. by using something like `<form netlify ... action="/success">`, then:
```

Comment:
I think this could be made much shorter like:

> If your Netlify form has a [custom success message](https://docs.netlify.com/forms/setup/#success-messages) like `<form netlify ... action="/success">` then ensure the corresponding `/routes/success.svelte` exists 

I don't think prerendering has to be enabled here

Reply:
I won't be able to reevaluate it until the weekend but it is (was?) definitely needed for whatever reason (see top of https://github.com/sveltejs/kit/issues/942#issuecomment-828726791).

Reply:
Ok. It's not mentioned in their docs, but if you tested it then that sounds sufficient. What do you think about making this section a single line like below

> If your Netlify form has a [custom success message](https://docs.netlify.com/forms/setup/#success-messages) like `<form netlify ... action="/success">` then ensure the corresponding `/routes/success.svelte` exists and is prerendered